WSLGd/main.cpp

```diff
@@ -301,7 +301,7 @@ try {
     THROW_LAST_ERROR_IF(chmod(c_dbusDir, 0777) < 0);
 
     std::filesystem::create_directories(c_x11RuntimeDir);
-    THROW_LAST_ERROR_IF(chmod(c_x11RuntimeDir, 0777) < 0);
```

Comment:
Should this be 01777 instead?

Reply:
@hideyukn88 - do you recall why 0777 was chosen?

Reply:
@benhillis, replied offline.

Reply:
thanks, looks like 777 is correct